data_steward/validation/main.py

```diff
@@ -615,7 +640,9 @@ def extract_date_from_rdr_dataset_id(rdr_dataset_id):
         # TODO remove dependence on date string in RDR dataset id
         rdr_date = rdr_date[:4] + '-' + rdr_date[4:6] + '-' + rdr_date[6:]
         return rdr_date
-    raise ValueError(f"{rdr_dataset_id} is not a valid rdr_dataset_id")
+    else:
```

Comment:
This is another higher level issue I'm glad you bring up: how to handle unset environment variables. Please put this in a separate branch and create a new ticket as we discussed.